notify/discord.py

```diff
@@ -0,0 +1,78 @@
+import json
+import logging
+import os
+import requests
+
+states = json.loads(os.environ["STATES"])
+
+
+def format_available_message(locations):
+    message = ":green_circle: Vaccine appointments available at {} location{}:".format(
+        "these" if len(locations) > 1 else "this",
+        "s" if len(locations) > 1 else "",
+    )
+    for location in locations:
+        if "earliest_appointment_day" in location:
+            if (
+                location["earliest_appointment_day"]
+                == location["latest_appointment_day"]
+            ):
+                day_string = " on *{}*".format(location["earliest_appointment_day"])
```

Comment:
Same with this formatting, does the markdown appear the same way as slack?
Example:
![](https://user-images.githubusercontent.com/5343931/109823900-f4446f00-7bfd-11eb-8661-68167f013cf2.png)